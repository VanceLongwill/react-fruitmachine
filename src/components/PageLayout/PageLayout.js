// @flow
import styled from 'styled-components'

export const PageLayout = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
`
