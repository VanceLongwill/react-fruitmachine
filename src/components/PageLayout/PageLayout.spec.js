import React from 'react'
import { mount } from 'enzyme'

import { PageLayout } from './PageLayout'

describe('PageLayout', () => {
  it('should match snapshot', () => {
    expect(mount(<PageLayout />)).toMatchSnapshot()
  })
})
