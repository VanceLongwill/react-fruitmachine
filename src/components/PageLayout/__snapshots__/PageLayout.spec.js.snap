// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageLayout should match snapshot 1`] = `
<styled.div>
  <div
    className="sc-bdVaJa kLZEci"
  />
</styled.div>
`;
